dace/config_schema.yml

```diff
@@ -38,23 +38,17 @@ required:
                     on compilation. Defaults to the transformation CLI, empty
                     string or an invalid class name skips the process.
 
-            visualize:
-                type: bool
-                default: false
-                title: Visualize SDFG
-                description: Open a GraphViz window after every transformation.
-
             visualize_sdfv:
                 type: bool
                 default: false
                 title: Visualize SDFG
                 description: Open a SDFV in browser every transformation.
 
-            savedots:
+            saveopts:
```

Comment:
Title is not so descriptive, maybe "save_intermediate"?

Reply:
I was thinking about it. Changing.